frontend/src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import { HStack, Icon, TooltipProps, VStack } from '@chakra-ui/react'
+import { Meta, Story } from '@storybook/react'
+
+import { viewports } from '~utils/storybook'
+
+import { Tooltip } from './Tooltip'
+
+export default {
+  title: 'Components/Tooltip',
+  component: Tooltip,
+  decorators: [],
+} as Meta
+
+const Template: Story<TooltipProps> = (args) => {
+  return (
+    // padding to create space for tooltips to appear
+    <VStack p="350px">
+      <Tooltip
+        {...args}
+        label="Display tooltip content right"
+        placement="right"
+      />
+      <Tooltip
+        {...args}
+        label="Display tooltip content left"
+        placement="left"
+      />
+      <Tooltip {...args} label="Display tooltip content top" placement="top" />
+      <Tooltip
+        {...args}
+        label="Display tooltip content bottom"
+        placement="bottom"
+      />
+    </VStack>
+  )
+}
+export const TooltipOnHover = Template.bind({})
+TooltipOnHover.args = {
+  children: <Icon name="Question" />,
+}
+
+const SingleToolTip: Story<TooltipProps> = (args) => {
+  return (
+    <HStack>
+      <div>OTP Verification Long Message</div>
+      <Tooltip {...args} />
+    </HStack>
+  )
+}
+
```

Comment:
Should we also have a story where the tooltip is already open (with `isOpen` prop)? For automated visual regression testing.